CMakeSettings.json

```diff
@@ -1,108 +1,126 @@
 ﻿{
-    "configurations": [
+  "configurations": [
```

Comment:
Should this file be updated manually for any new option added to cmake?

Reply:
Yes, unfortunately this file is manually edited. It is only used for the Visual Studio 2019 IDE builds. I would like to add a few predefined configurations. But we need to do a small write-up how customers may edit this to their own liking.